### DISCUSSION TITLE
Clustering different tissues

### DISCUSSION THREAD

**Discussion by mbartl13** [satijalab/seurat#5483] [Q&A]
Hi all! I have data from PBMCs and lymph node from the same individuals. Can I cluster these together or should I cluster each tissue independently?

**Comment by samuel-marsh** [comment]
Hi,

Unfortunately no right answer here.  That is going to completely depend on individual experimental variables of your specific samples and what the goal of your analysis is.  I suggest trying it out multiple ways to see what works best in your individual scenario.

Best,
Sam

**Comment by mbartl13** [comment]
Sounds good, thanks Sam!
________________________________
